verify/install-service

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -eufx
+
+SECRETS=/opt/cockpit-verify-secrets
+IMAGES=/opt/cockpit-verify-images
+
+echo "options kvm-intel nested=1" > /etc/modprobe.d/kvm-intel.conf
+echo "options kvm-amd nested=1" > /etc/modprobe.d/kvm-amd.conf
+( rmmod kvm-intel && modprobe kvm-intel ) || ( rmmod kvm-amd && modprobe kvm-amd )
```

Comment:
Moved it to the end of the script, so it will fail the script, but the remainder will have been configured appropriately. Lets see if that works in practice.